seneca.js

```diff
@@ -2151,6 +2153,19 @@ function make_seneca( initial_options ) {
   }
 
 
+  // Inspired by https://github.com/hapijs/hapi/blob/master/lib/plugin.js decorate
+  function api_decorate(property, method) {
+    assert.ok(property, 'property must be specified')
```

Comment:
typical use case would be a plugin adding decorations?
in which case, failure is fatal as the plugin will not init properly

in general we have to think more clearly about seneca error handling - microservices need to fail fast, but also need to announce that they have failed, so it's not enough in general to just fail on the normal way